README.md

```diff
@@ -21,133 +21,82 @@ $ cargo install cargo-ebuild
 
 ## Usage
 
-You will first need to get the sources to the crate you want to install.
+Firstly, you need to get the sources to the crate you want to install.
 Your best bet is to search for the crate at [crates.io](https://crates.io)
 and follow the *Repository* link. This should give you the ability to clone
 the repo. Once you have cloned the repo, change into the directory and
 ensure that you checkout the tag corresponding to the version you would like
-to package. Lastly you will execute the `cargo ebuild` command to generate the
-ebuild for that crate.
+to package. The package must contain `Cargo.lock` file to resolve dependency
+versions, otherwise you need to build the project so Cargo can generate it.
```

Comment:
So you don't really need to build the package. You can simply do `cargo generate-lockfile`. We can probably even do it within the ebuild command.

Reply:
I think it is a good solution, it can be done when `Cargo.lock` is not found instead of giving an error message.

Reply:
By the way, it doesn't need to check at all, let's just wire `cargo generate-lockfile` as it will check it by itself.